api/routes.go

```diff
@@ -91,6 +91,8 @@ func addRoutes(r *gin.Engine, auth Authentication, tokenHandler TokenHandler, uc
 	router.GET("/scenario-publications", tom, handleListScenarioPublications(uc))
 	router.POST("/scenario-publications", tom, handleCreateScenarioPublication(uc))
 	router.POST("/scenarios-testrun", tom, handleCreateScenarioTestRun(uc))
+	router.GET("/scenario-testrun", tom, handleListScenarioTestRun(uc))
```

Comment:
cosmetic only request: please move the 3 endpoints related to test runs out of the bloc of endpoints related to scenario-publications (we can have another "bloc" just below)